libraries/app/include/graphene/app/api.hpp

```diff
@@ -464,7 +464,8 @@ namespace graphene { namespace app {
          vector<asset_holders> get_all_asset_holders() const;
 
       private:
-         graphene::chain::database& _db;
```

Comment:
Perhaps better keep this reference and assign+validate in constructor so you don't have to check in each api call.

Reply:
Its done. Please review